tests/installation/bootloader_s390.pm

```diff
@@ -73,6 +73,35 @@ sub create_infofile {
     return shorten_url(autoinst_url . "/files/$path");
 }
 
+sub prepare_agama_boot_params {
+    my @params = ();
+
+    # add mandatory boot params
+    push @params, 'cio_ignore=all,!condev,!0.0.0150';
+    push @params, 'rd.zdev=dasd,0.0.0150' if (get_var('S390_DISK') ne 'ZFCP');
+    push @params, 'hvc_iucv=8';
+    push @params, "live.password=$testapi::password";
+
+    # override default boot params
+    if (get_var('BOOTPARAMS')) {
+        push @params, split ' ', trim(get_var('BOOTPARAMS'));
+        return @params;
+    }
+
+    # add default boot params
+    if (my $inst_auto = get_var('INST_AUTO')) {
+        my $profile_url = ($inst_auto =~ /\.libsonnet/) ?
+          generate_json_profile($inst_auto) :
+          expand_agama_profile($inst_auto);
+        set_var('INST_AUTO', $profile_url);
+        push @params, "inst.auto=\"$profile_url\"";
+        push @params, "inst.finish=stop";
```

Comment:
I think we don't need add init.finish here for s390x, that already added in lib/bootloader_setup.pm:898